### DISCUSSION TITLE
Error response 500

### DISCUSSION THREAD

**Discussion by eddcole** [Blackymas/NSPanel_HA_Blueprint#312] [Q&A]
Hi all, i cannot save any profile as i keep getting error respose 500. Can anybody tell me what im doing wrong. Also my device will not go past the please wait screen but shows the IP. All the buttons and sensors are coming up and appear to be working.

**Comment by Blackymas** [comment]
you have create a issue

https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/310
